app/src/main/sqldelight/ca/llamabagel/transpo/data/db/Route.sq

```diff
@@ -7,15 +7,21 @@ CREATE TABLE route (
     color TEXT NOT NULL
 );
 
-insert:
-INSERT INTO route(id, short_name, long_name, type, service_level, color)
-VALUES (?, ?, ?, ?, ?, ?);
+CREATE VIRTUAL TABLE route_fts USING fts4 (
+    id TEXT NOT NULL PRIMARY KEY,
+    short_name TEXT NOT NULL,
+    long_name TEXT,
+    type INTEGER As Integer NOT NULL,
+    service_level TEXT NOT NULL,
+    color TEXT NOT NULL
+);
 
 deleteAll:
-DELETE FROM route;
+DELETE FROM route_fts;
```

Comment:
Do we not still need a function to delete all records from the non-fts table? Does deleting in the fts table delete the real record too?

Reply:
Nothing is being stored in the non-fts table. We only insert into the fts table.

Reply:
I see, interesting. Could  this be applied to all tables that we're using (for searchable stuff at least)

Reply:
I couldn't get that to work for the `Stop` table. I don't think fts tables can handle `as StopId` casting.